.htmllintrc

```diff
@@ -0,0 +1,15 @@
+{
+ "attr-name-ignore-regex": false,
+ "attr-name-style":["lowercase"],
```

Comment:
It's not in the manual https://github.com/htmllint/htmllint/wiki/Options#attr-name-style
![attr](https://cloud.githubusercontent.com/assets/12227945/22110197/d6d243e4-de80-11e6-8fc7-b1de51ae42fd.png)